tasks/release-notes/README.md

```diff
@@ -0,0 +1,10 @@
+# Release Notes
+
+This script builds release notes for a milestone:
+
+```
+yarn build:release-notes v0.34.0
+# there should be a new file at ./tasks/release-notes/v0.34.0-release-notes.md
+```
+
+To run this script, you'll need a personal access token. Provision one from https://github.com/settings/tokens and set it to `GITHUB_TOKEN` in your env.
```

Comment:
what permissions do I need? Also, since I nuke the local codebase with git clean -fxd all the time, I'll likely just add to my local shell. fwiw

Reply:
Ah I didn't mean make an actual env file, but have it in your bashrc or zshrc. I can make that more clear. Is that something you can do / that's reasonable? Otherwise we get an authentication error